test/activityrepo-test.js

```diff
@@ -237,18 +237,15 @@ describe('Activity', function() {
     ]);
   });
 
-  it.only('should return the highest number of stairs climbed in a day for all time', function() {
+  it('should return the highest number of stairs climbed in a day for all time', function() {
     expect(activity.getStairRecord(11)).to.eql(33);
   });
-
-  it.only('should show a 3-day increasing streak for a users step count', function () {
+/* TESTS ARE FAILING */
```

Comment:
@JoshSevy I meant to ask you about these too. Was this one of the situations where the class couldn't see the userRepo?

Reply:
@GreysonElkins I got them resolved yesterday and am about to put in a pull request that has these methods refactored to the new methods in Repo. All test are passing on my end.